libs/sdk-ui-charts/src/highcharts/chartTypes/_chartCreators/customConfiguration.ts

```diff
@@ -892,26 +921,31 @@ function getHoverStyles({ type }: any, config: any) {
         case VisualizationTypes.PIE:
         case VisualizationTypes.DONUT:
         case VisualizationTypes.TREEMAP:
-            seriesMapFn = (seriesOrig) => {
-                const series = cloneDeep(seriesOrig);
-
+            seriesMapFn = (series) => {
                 return {
                     ...series,
                     data: series.data.map((dataItemOrig: any) => {
-                        const dataItem = cloneDeep(dataItemOrig);
-                        const drilldown = dataItem?.drilldown;
-
-                        set(
-                            dataItem,
-                            "states.hover.brightness",
-                            drilldown ? HOVER_BRIGHTNESS : MINIMUM_HC_SAFE_BRIGHTNESS,
-                        );
-
-                        if (!drilldown) {
-                            set(dataItem, "halo.size", 0); // see plugins/pointHalo.js
-                        }
-
-                        return dataItem;
+                        const drilldown = dataItemOrig?.drilldown;
+                        const pointHalo = !drilldown
+                            ? {
+                                  halo: {
+                                      ...dataItemOrig?.halo,
+                                      size: 0,
+                                  },
+                              }
+                            : {}; // see plugins/pointHalo.js
```

Comment:
The comment used to be in the `size: 0` branch, is the move here intentional?

Reply:
Not intentional, my mistake, I've put it to the size :0 branch